cogs/repmod_moodle.py

```diff
@@ -245,13 +245,25 @@ def file_report(self, usr, grade, comment):
         messages.append(msg)
         logger.info(self._format_msg(msg))
 
-        # Check Moodle User
-        if usr['auth'] != 'moodle':
-            msg = "repmod_moodle: Requires user with authmod 'moodle'"
+        # Check Moodle or LDAP User
```

Comment:
I think we can probably remove this check all together now that we just lookup usernames. But again, not a huge issue.